src/pages/settings/Wallet/WalletEmptyState.js

```diff
@@ -35,7 +34,7 @@ function WalletEmptyState({onAddPaymentMethod}) {
     const {translate} = useLocalize();
     return (
         <IllustratedHeaderPageLayout
-            backgroundColor={themeColors.PAGE_BACKGROUND_COLORS[SCREENS.SETTINGS.WALLET]}
+            backgroundColor={colors.green700}
```

Comment:
Let's not import colors.js and instead use the previous pattern for colors here :) (similar to the workspaces page). 

`themeColors.PAGE_BACKGROUND_COLORS[SCREENS.SETTINGS.WALLET]`